cmd/bf-ia-broker/landsatMetadata.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"database/sql"
+	"log"
+
+	"github.com/venicegeo/bf-ia-broker/landsat_localindex/db/metadata"
+	"github.com/venicegeo/bf-ia-broker/util"
+	cli "gopkg.in/urfave/cli.v1"
+)
+
+const missingDataQuery = `
+SELECT product_id, scene_url FROM scenes WHERE 
+corner_ul IS NULL OR 
+corner_ur IS NULL OR  
+corner_ll IS NULL OR 
+corner_lr IS NULL 
+`
+
+const insertSQL = `
+UPDATE scenes SET 
+	corner_ul = st_setSRID( st_MakePoint($2, $3), 4326), 
+	corner_ur = st_setSRID( st_MakePoint($4, $5), 4326),
+	corner_lr = st_setSRID(st_MakePoint($6, $7), 4326),
+	corner_ll = st_setSRID( st_MakePoint($8, $9), 4326)
+WHERE product_id = $1
+`
+
+type sceneRow struct {
+	productID string
+	url       string
+}
+
+type sceneMetadata struct {
+	productID string
+	metadata  *metadata.LandsatSceneMetadata
+}
+
+func landsatPopulateMetadata(*cli.Context) {
+	conn, err := getDbConnection(&util.BasicLogContext{})
+
+	if err != nil {
+		log.Fatal("Error opening db connection.")
+	}
+	defer conn.Close()
+
+	//Create the statement to inser the info into the database.
+	insertStmt, err := conn.Prepare(insertSQL)
+	if err != nil {
+		log.Fatal("Error preparing insert statement: " + err.Error())
+	}
+	defer insertStmt.Close()
+
+	rows, err := conn.Query(missingDataQuery)
+	if err != nil {
+		log.Fatal("Error with query.")
+	}
+	defer rows.Close()
+
+	numWorkers := 10
```

Comment:
Should this be configurable?

Reply:
Maybe. Using workers is just to get a minor speed-up by doing some of the HTTP queries in parallel, and the footprint should be small enough that the difference between 10 and 1 is trivial. Pushing the worker count higher will also only gain little more as we run into bandwidth limits.

@akater320 thoughts?